folium/features.py

```diff
@@ -616,6 +669,23 @@ def __init__(self, locations, color=None, weight=None,
                 {{this._parent.get_name()}}.addLayer({{this.get_name()}});
             {% endmacro %}
             """)  # noqa
+    def _get_self_bounds(self):
+        """Computes the bounds of the object itself (not including it's children)
+        in the form [[lat_min, lon_min], [lat_max, lon_max]]
+        """
+        bounds = [[None,None],[None,None]]
+        for point in iter_points(self.data):
+            bounds = [
+                [
+                    none_min(bounds[0][0], point[0]),
```

Comment:
I see what you are trying to do here (and I do not have a better suggestion right now), but it looks like we could do something simpler...